src/Stateless/StateRepresentation.cs

```diff
@@ -45,18 +45,18 @@ internal ICollection<StateRepresentation> GetSubstates()
                 return _substates;
             }
 
-            public bool CanHandle(TTrigger trigger)
+            public bool CanHandle(TTrigger trigger, params object[] args)
             {
-                return TryFindHandler(trigger, out TriggerBehaviourResult unused);
+                return TryFindHandler(trigger, args, out TriggerBehaviourResult unused);
             }
 
-            public bool TryFindHandler(TTrigger trigger, out TriggerBehaviourResult handler)
+            public bool TryFindHandler(TTrigger trigger, object[] args, out TriggerBehaviourResult handler)
             {
-                return (TryFindLocalHandler(trigger, out handler) ||
-                    (Superstate != null && Superstate.TryFindHandler(trigger, out handler)));
+                return (TryFindLocalHandler(trigger, args, out handler) ||
+                    (Superstate != null && Superstate.TryFindHandler(trigger, args, out handler)));
             }
 
-            bool TryFindLocalHandler(TTrigger trigger, out TriggerBehaviourResult handlerResult)
+            bool TryFindLocalHandler(TTrigger trigger, object[] args, out TriggerBehaviourResult handlerResult)
             {
                 if (!_triggerBehaviours.TryGetValue(trigger, out ICollection<TriggerBehaviour> possible))
                 {
```

Comment:
I get a compiler error on line 69, (h.UnmetGuardConditions):
Error	CS1503	Argument 2: cannot convert from 'method group' to 'ICollection<string>'	Stateless(net40), Stateless(net45), Stateless(netstandard1.0)	C:\Users\Henning.Torsteinsen\Source\Repos\trgrote-stateless\src\Stateless\StateRepresentation.cs	Active	69